extensions/telemetry/package.json

```diff
@@ -12,12 +12,15 @@
     "build": "webpack --config webpack.config.js",
     "dev": "npm run build --watch"
   },
-  "dependencies": {},
+  "dependencies": {
+  },
   "devDependencies": {
     "ts-loader": "^8.0.4",
     "typescript": "^4.0.3",
     "webpack": "^4.44.2",
     "mobx": "^5.15.5",
-    "react": "^16.13.1"
+    "react": "^16.13.1",
+    "node-machine-id": "^1.1.12",
```

Comment:
Should this be under `dependencies` instead?

Reply:
I thought we can bundle it using webpack?

Reply:
If this is something used in main/node then it's better to keep it as normal npm-package instead of bundling?

Reply:
It's used on both sides (main and renderer). Does that make webpack bundling more preferrable?